contributing/code/standards.rst

```diff
@@ -100,6 +100,9 @@ Structure
 
 * Place unary operators (``!``, ``--``, ...) adjacent to the affected variable;
 
+* Use `Yoda conditions`_ when checking a variable against an expression to avoid
```

Comment:
Please consider my comment: https://github.com/symfony/symfony-docs/pull/5402#discussion_r33360159

Is contributors are agree with it of course.